### ISSUE TITLE
Type 'T' exists in both 'A.dll' and 'B.dll'

### ISSUE THREAD

**Issue by SlowLogicBoy** [filipw/dotnet-script#206]
```csharp
#! "netcoreapp2.0"
#r "nuget:Nuget.Client,*"

using System.Xml.Linq;

var xml = XDocument.Load("somefile");
```

d:\Code\Git\dnScripts\helloworld.csx(6,11): error CS0433: The type 'XDocument' exists in both 'System.Private.Xml.Linq, Version=4.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' and 'System.Xml.XDocument, Version=4.0.11.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'

Works on .csproj, script generated .csproj is also identical.

I'm using https://ci.appveyor.com/project/filipw/dotnet-script/build/1.0.295 build (latest master build)

**Comment by seesharper** [comment]
This works in the bugfix/core-assemblies branch. If I can get the darn build to run on Travis :) Working on it :)

**Comment by SlowLogicBoy** [comment]
I will test it out once it's built.

**Comment by seesharper** [comment]
Fixed by #205
